examples/ping.rs

```diff
@@ -75,17 +74,15 @@ fn main() -> Result<(), Box<dyn Error>> {
         println!("Dialed {}", addr)
     }
 
-    block_on(future::poll_fn(move |cx| loop {
-        match swarm.poll_next_unpin(cx) {
-            Poll::Ready(Some(event)) => match event {
+    block_on(async move {
```

Comment:
Can we also use an `async fn main` in this example?

Reply:
That's what I was thinking as well, but I think we need to mark the `async fn main` with an attribute, which means we either have to use `tokio` or `async_std`, is that right?

Reply:
Yes, I think it is fine. I suggest using `async-std` in all cases apart from the tokio specific ones. Not that I prefer `async-std` over `tokio`, but simply for consistency, given that `async-std` is the default across the whole repository.